images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -0,0 +1,235 @@
+function Get-OSName {
+    $systemInfo = Get-CimInstance -ClassName Win32_OperatingSystem
+    $OSName = $systemInfo.Caption
+    return $OSName
+}
+
+function Get-OSVersion {
+    $systemInfo = Get-CimInstance -ClassName Win32_OperatingSystem
+    $OSVersion = $systemInfo.Version
+    $OSBuild = $systemInfo.BuildNumber
+    return "OS Version: $OSVersion Build $OSBuild"
+}
+
+function Get-JavaVersionsList {
+    param(
+        [string] $DefaultVersion
+    )
+
+    $postfix = ""
+    $javaDir = Join-Path $env:PROGRAMFILES "Java"
+    return Get-ChildItem $javaDir | ForEach-Object {
+        $javaBinPath = Join-Path $_ "bin"
+        $rawVersion = & cmd /c "`"$javaBinPath\java.exe`" -version 2>&1" | Out-String
+        $rawVersion -match 'openjdk version "(?<version>.+)"' | Out-Null
+        $version = $Matches.Version
+        if ($version -match $DefaultVersion) {
+            $postfix = "(default)"
+        } else {
+            $postfix = ""
+        }
+        return "Java $version $postfix"
+    } | Sort-Object {
+        $version = ($_.Split(" ")[1]).Split("_")[0]
+        return [System.Version]$version
+    }
+}
+
+function Get-RustVersion {
+    $rustVersion = [regex]::matches($(rustc --version), "\d+\.\d+\.\d+").Value
+    return "Rust ${rustVersion}"
+}
+
+function Get-PythonVersion {
+    return $(python --version)
```

Comment:
Why don't we use simple statement?

```
function Get-PythonVersion { 
    python --version
}
```

Reply:
or `& python --version`.
`$()` is not needed

Reply:
The same for other places

Reply:
Done